README.md

```diff
@@ -12,6 +12,7 @@ Detailed instructions are available in our Installation Guides below.
 
 Sumo Logic Helm Chart Version
 |:--------
+[0.17.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.17.0/deploy/README.md)
 [0.16.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.16.0/deploy/README.md)
 [0.15.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.15.0/deploy/README.md)
 [0.14.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.14.0/deploy/README.md)
```

Comment:
@perk-sumo @samjsong  I was thinking maybe we should just list the last 5 versions here so, until 0.13?

Reply:
I think it's the way to go, let's consult that with @breinero and push as a fix to this branch later on.